src/tests/ftest/util/job_manager_utils.py

```diff
@@ -373,17 +373,22 @@ def assign_environment_default(self, env_vars):
         """
         self.export.update_default(env_vars.to_list())
 
-    def run(self):
+    def run(self, raise_exception=None):
```

Comment:
None is not a bool

Reply:
Right. But when something IS passed, it's a `bool`. I would think the default here should be `False`, but out of the scope of this PR